src/display/api.js

```diff
@@ -1412,6 +1420,7 @@ class PDFPageProxy {
     annotationMode = AnnotationMode.ENABLE,
     transform = null,
     canvasFactory = null,
+    filterFactory = null,
```

Comment:
I don't think we need/want to add a `filterFactory` parameter here, since the idea is to (in the future) remove the now deprecated `canvasFactory` one